app/Models/AchievementSet.php

```diff
@@ -37,6 +37,7 @@ class AchievementSet extends BaseModel
         'user_id',
         'players_total',
         'players_hardcore',
+        'achievements_published_at',
```

Comment:
nit: Recommend renaming this column to `achievements_first_published_at`. Initially, I erroneously assumed this would update every time an achievement was published to the set.

Reply:
Done. You'll want to manually rename the column in your database. It's much faster than doing a `migrate:rollback`/`migrate`, and you won't lose any data you may have already aggregated.